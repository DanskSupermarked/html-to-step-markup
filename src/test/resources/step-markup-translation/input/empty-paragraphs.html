<P><STRONG>Fodboldtrøje fra Kappa i høj italiensk kvalitet og smart snit med mange detaljer. </STRONG></P>
<P> </P>
<P>Designet er baseret på den orginale Kappa Kombat Jersey, som benyttes bl.a. af FCK og andre internationelle fodboldklubber.</P>
<P> </P>
<P>Fodboldtrøjen er med korte ærmer og har print på ærmerne. Trøjen har endvidere quick dry egenskab, der gør at trøjen tørre hurtigt. </P>
