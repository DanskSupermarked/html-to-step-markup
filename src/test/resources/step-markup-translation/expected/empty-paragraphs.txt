<bold>Fodboldtrøje fra Kappa i høj italiensk kvalitet og smart snit med mange detaljer.</bold><return/>
<return/>
Designet er baseret på den orginale Kappa Kombat Jersey, som benyttes bl.a. af FCK og andre internationelle fodboldklubber.<return/>
<return/>
Fodboldtrøjen er med korte ærmer og har print på ærmerne. Trøjen har endvidere quick dry egenskab, der gør at trøjen tørre hurtigt.
